client_test/rpc_test.go

```diff
@@ -120,7 +120,7 @@ func setupNitroNodeWithRPCClient(
 	messageservice := p2pms.NewMessageService("127.0.0.1",
 		msgPort,
 		crypto.GetAddressFromSecretKeyBytes(pk),
```

Comment:
By allowing the SC address to be passed in separately from the `pk`,  it allows the option of using a different message key than the SC key.